package.json

```diff
@@ -7,27 +7,28 @@
   "license": "Apache",
   "private": true,
   "scripts": {
-    "prebuild": "yarn clean",
+    "prebuild": "npm run clean",
     "dr:surge": "node dr-surge.js",
     "build": "NODE_ENV=production webpack --config ./config/webpack.prod.js",
-    "build:remote": "DATA_SOURCE=remote yarn build",
-    "build:mock": "DATA_SOURCE=mock yarn build && yarn dr:surge",
+    "build:remote": "DATA_SOURCE=remote npm run build",
+    "build:mock": "DATA_SOURCE=mock npm run build && npm run dr:surge",
     "setup:dev:remote": "node ./scripts/remote-dev.js",
     "start": "node deploy/server.js",
-    "start:remote": "DATA_SOURCE=remote yarn start",
-    "start:mock": "DATA_SOURCE=mock yarn start",
+    "start:remote": "DATA_SOURCE=remote npm run start",
+    "start:mock": "DATA_SOURCE=mock npm run start",
     "start:dev": "./scripts/start-dev.sh",
-    "start:dev:remote": "yarn setup:dev:remote && DATA_SOURCE=remote yarn start:dev",
-    "start:dev:mock": "DATA_SOURCE=mock yarn start:dev",
+    "start:dev:remote": "npm run setup:dev:remote && DATA_SOURCE=remote npm run start:dev",
+    "start:dev:mock": "DATA_SOURCE=mock npm run start:dev",
     "test": "jest",
     "eslint": "eslint --ext .tsx,.js ./src/",
```

Comment:
I think we need `$(npm bin)/eslint` here as well, although it appears to be working as-is, not sure why that is